src-docs/src/views/modal/modal_example.js

```diff
@@ -38,9 +38,6 @@ const confirmModalSource = require('!!raw-loader!./confirm_modal');
 import ConfirmLoadingModal from './confirm_modal_loading';
 const confirmModalLoadingSource = require('!!raw-loader!./confirm_modal_loading');
 
-import ModalWidth from './modal_width';
-const modalWidthSource = require('!!raw-loader!./modal_width');
```

Comment:
Since we're removing this, could we also delete the file?

Reply:
Perfect, I will do!

Reply:
Updated changes @BSFishy, although there is an issue for testing commits with only deletes.

Reviewing the base code, I realised that when you commit new changes, a script runs before the commit (it makes use of **pre-commit** library) to test files in the staging area. This script is defined in the file `test-staged.js` under the `./script folder`. However, the stagedFiles captured by this script (variable `stagedFiles`) using the command execSync('git diff --cached --name-only --diff-filter=ACMR') only considers files that are _Added_, _Copied_, _Modified_ or _Renamed_, but never a _Deleted_. So when I try to commit a change that **only** has a _Delete_ the variable `stagedFiles` is an empty list as `['']` and then the command from jest with flag `--findRelatedTests` complains because no location is given.

Reply:
Yep, that would be a bug in `scripts/test-staged.js`

Reply:
Issue #1071 was created for this error. I will take a look to it.